src/main/resources/application.conf

```diff
@@ -0,0 +1,25 @@
+# locations
+bucket_name = "mybucket"
+bucket_landing_path = /landing
+bucket_staging_path = /staging
+bucket_merge_path = /merge
+
+# merging params
+merge_frequency = 60 seconds
+merge_min_files = 2
+merge_factor = 10000
+landing_purge_tolerance = 4 seconds
+
+# storage params
+s3_ssl_enabled = false
+s3_endpoint = "127.0.0.1:8000"
+s3_access_key = "accessKey1"
+s3_secret_key = "verySecretKey1"
+
+# event data source
+kafka_bootstrap_servers = "kafka:9092"
+kafka_topic = "backbeat-clueso"
+
+# spark execution params
+trigger_time = 10 seconds
+checkpoint_path = ./_checkpoint
```

Comment:
Apparently there is no good docker swarm solution (yet) for floating containers with volumes but we do need to have a volume for the checkpoint directory so that we are not filling up the disk partition where items in the container write when there is no volume. So, we should create a volume on each possible machine where a worker could be. We will then need to test whether a new container starting up where another had been writing to the checkpoint does not cause an issue. If it does, we will have to get clever about how we write to subdirectories in /_checkpoint.